src/components/AdminPane/HOCs/WithChallengeManagement/WithChallengeManagement.js

```diff
@@ -163,6 +163,11 @@ async function uploadLineByLine(
       taskLines = _compact(taskLines);
     }
 
+    if (taskLines.length === 0) {
```

Comment:
what if taskLines.length has no value?  is that a possibility?

Reply:
I dont think so, but i will add fallbacks incase that happens